dogechain-lab/dogechain
# Update precompile contracts

# Description

This PR upgrades the system pre-compiled contracts to the latest version, which is audited.

The commit link: https://github.com/dogechain-lab/jury-contracts/commit/b1387d0030f8911465a33ce6aa84e87fd4277ae1

# Changes include


# Breaking changes

The `ValidatorSet` contract slots are not backward compatible, so we have to redeploy the network.

# Checklist


## Testing


### Manual tests

* Deploy a dev mode server.
* Connect the validator with remix.
* Check all the contract reading ABIs.
